react-context/client/src/components/NavBar.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useContext } from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import { FaBolt, FaSignOutAlt } from 'react-icons/fa';
+import { AppContext } from './AppContext';
+
+export function NavBar() {
+  useContext(AppContext);
+
+  /* TODO: Grab `user` and `handleSignOut` from `AppContext` */
+  const user = useContext(AppContext).user;
+  const handleSignOut = useContext(AppContext).handleSignOut;
```

Comment:
The easiest way to get these is with destructuring:
`const { user, handleSignOut } = useContext(AppContext);`